File: SWTOR HoloNet/SWTOR HoloNet/Module/Forum/Model/ForumCategory.swift
//
//  ForumCategory.swift
//  SWTOR HoloNet
//
//  Created by Ivan Fabijanovic on 16/10/14.
//  Copyright (c) 2014 Ivan Fabijanović. All rights reserved.
//

import UIKit

class ForumCategory: Entity {
    
    // MARK: - Properties
    
    var iconUrl: String?
    var title: String
    var description: String?
    var stats: String?
    var lastPost: String?
    
    // MARK: - Init
    
    init(id: Int, title: String) {
        self.title = title
        super.init(id: id)
    }
    
}

File: android/app/src/main/java/com/ifabijanovic/holonet/forum/model/ForumCategory.kt
package com.ifabijanovic.holonet.forum.model

import com.ifabijanovic.holonet.app.model.Entity

/**
 * Created by feb on 19/03/2017.
 */
data class ForumCategory(
        override val id: Int,
        val iconUrl: String?,
        val title: String,
        val description: String?,
        val stats: String?,
        val lastPost: String?
): Entity {}
